content/functions/title.md

```diff
@@ -23,3 +23,12 @@ aliases: []
 ```
 {{title "BatMan"}}` → "Batman"
 ```
+
+Can be piped to combine with others.
+
+```
+{{ range $name, $items := .Site.Taxonomies.<YOUR_TAXONOMY> }}
+  <li><a href="{{ $.Site.BaseURL }}<YOUR_TAXONOMY>/{{ $name | urlize | lower }}">{{ $name  | humanize | title }} ({{ len $items }}</a>
```

Comment:
Definitely in the docs since that's the first place users will look for what's considered the "best practice" for general Hugo usage...